### PR TITLE
Add prizes page

### PR DESCRIPTION
**Pull request by jamieliu386**
Fixes: #33 

Adds a prize page containing a table with tracks and prizes. Images for prize tracks are hidden if the screen width is too small.

### CODE REVIEW

File: src/components/PrizePage/PrizeHeader.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	const classes = useStyles();
+
+	return (
+		<Box display='flex' justifyContent='center' alignItems='center'>
+			<SvgImg src={wrench} className={classes.wrench} height={39} width={62} />
+			<Typography variant='h3' component='h2' className={classes.title}>
+				Prize Tracks
+			</Typography>
+			<SvgImg src={wrench} className={classes.wrench} height={39} width={62}
+				style={{ transform: 'rotate(180deg)' }} />
```
Comment (jamieliu386) [inline src/components/PrizePage/PrizeHeader.js:33]: The wrenches are positioned weirdly when the screen width is low enough to make "Prize Tracks" wrap to two lines, not sure how to fix.

Also, I feel like the wrenches look out of place in the design of the website... thoughts?

File: src/components/PrizePage/Prizes.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import DesignHack from './images/design-hack.png';
+import GameHack from './images/game-hack.png';
+import MLHack from './images/ml-hack.png';
+import MobileHack from './images/mobile-hack.png';
+import OverallHack from './images/overall-hack.png';
+import WebHack from './images/web-hack.png';
+
+const allPrizes = [
```
Comment (jamieliu386) [inline src/components/PrizePage/Prizes.js:23]: Also, I used the prizes from the Figma but I will change this to match the prizes we actually gave out for HOTH lol

File: src/components/PrizePage/Prizes.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+import useTheme from '@material-ui/core/styles/useTheme';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import DesignHack from './images/design-hack.png';
+import GameHack from './images/game-hack.png';
+import MLHack from './images/ml-hack.png';
+import MobileHack from './images/mobile-hack.png';
+import OverallHack from './images/overall-hack.png';
+import WebHack from './images/web-hack.png';
```
Comment (jamieliu386) [inline src/components/PrizePage/Prizes.js:21]: I got these off of the Figma, but perhaps we can design our own icons for the prize tracks? Or look for higher quality images?

File: src/components/PrizePage/PrizeHeader.js (hunk 1 above)
Comment (Timthetic) [reply src/components/PrizePage/PrizeHeader.js:33]: Screen Shot 2020-06-23 at 12 47 32 AM
Here is Google Chrome pretending to be an iPhone 6. I think this looks pretty good on mobile. My first instinct would be not to worry about it.
I don't think the wrenches look out of place, but I also know 0 about style.

**Comment by jamieliu386** [comment]
Ok great, I'll merge for now and we can tweak the design/images later as necessary!

**Pull request merged**